inc/alert.class.php

```diff
@@ -52,7 +52,9 @@ class Alert extends CommonDBTM {
    const ACTION      = 5;
    const PERIODICITY = 6;
 
-
+   /**
+    * @see CommonDBTM::prepareInputForAdd($input)
```

Comment:
Indeed doc is not up to date on this point; I'll fix it; that is definitely not the way to go for generated docs.

There are 2 ways to have an method phpdoc with a correct output:
1- do not add docblock on child methods that does not change parent behavior,
2- add the full docblok each time.

I do not like the second solution very much, since we'll have to update child methods docs every time parent method change.

I'd love to find something that would be easy to maintain (no copy-paste), easy to read (every method documented in any way, overridden or not) and phpdoc/apigen compliant... But as far as I can see (refer to tests outputs); I'm not sure this is actually possible.